airbyte-cdk/python/bin/migrate_options_to_parameters.sh

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env bash
+
+echo "Starting migration"
+
+[ -z "$ROOT_DIR" ] && exit 1
+
+CONNECTORS_DIR=$ROOT_DIR/airbyte-integrations/connectors
+CDK_DIR=$ROOT_DIR/airbyte-cdk/python/
+
+for directory in $CONNECTORS_DIR/source-* ; do
+  MANIFEST_DIRECTORY=$(basename $directory | tr - _)
+  SOURCE_NAME=${MANIFEST_DIRECTORY#source_}
+  FILEPATH=$directory/source_$SOURCE_NAME/$SOURCE_NAME.yaml
+
+  echo "Migrating manifest located at $FILEPATH"
+  if test -f $FILEPATH; then
+    # In place replacement of $options to $parameters
+    sed -i '' -E 's/\$options/\$parameters/' $FILEPATH
```

Comment:
Fair point, I can kind of see it going either way. It might overdoing it to make a really generic search replace for what might be a short term set of simple migrations. But might revisit this if it starts to get unwieldly